src/main/index.ts

```diff
@@ -146,9 +158,19 @@ if (!isFirstInstance) {
   };
 
   app.on('session-created', (session) => {
+    let reservedStateValues = [];
     if (!isDevelopment) {
       log.info('Cleaning local storage on session creation');
-      session.clearStorageData({ storages: ['localstorage'] }).catch(log.error);
+
+      if (mainWindow != null) {
+        mainWindow.webContents.executeJavaScript('({...localStorage});', true).then((localStorage) => {
+          reservedStateValues = reservedStates.map((state) => localStorage.getItem(state));
```

Comment:
You are only assigning `reservedStateValues` here, and using it inside this function scope. Why do you need to declare `reservedStateValues` outside this function?

Reply:
This is not the `localStorage` object in the renderer process, I don't think .getItem() exists - have you verified this code works?